include/openmc/distribution_spatial.h

```diff
@@ -37,6 +37,24 @@ class CartesianIndependent : public SpatialDistribution {
   UPtrDist z_; //!< Distribution of z coordinates
 };
 
+//==============================================================================
+//! Distribution of points specified by independent distributions in
+//r,theta,phi
```

Comment:
We use Doxygen to style our comments like these. In this case, you can fit the entire sentence on one line (and still keep it under 80 charactrers) which will make Doxygen happy.

But for future reference: every line of the comment needs to start with `//! `. Also it's usually best to try making a one-line description of a class so Doxygen will interpret it as a "brief" description. (A longer description can be added after a blank line. See neighbor_list.h for an example.)